README.md

```diff
@@ -102,17 +104,49 @@ This code pattern was tested with HDP Search (`Solr`) v6.6.2 and assumes that it
 
    1. Run the `solr status` command:
       ```
-      cd <solr_installation_dir>/solr/bin
+      cd <solr_installation_dir>/bin
       ./solr status
       ```
 
-2. Install the `Solr vector scoring plugin` using the following [instructions](https://github.com/saaay71/solr-vector-scoring). 
+4. Install the `Solr vector scoring plugin`.
 
-    > Note: When making changes to the `solrconfig.xml` and `managed_schema` files, avoid using cut and paste to reduce the chance of unwanted characters being inserted.
+    The installation instructions and plugin jar file can be found in this [repository](https://github.com/saaay71/solr-vector-scoring). 
 
-    To test out your changes, try out the samples shown in the [Examples](https://github.com/saaay71/solr-vector-scoring#example) section of the plugin's README. This will require you to create test collections - see the [Running Solr Docs](https://lucene.apache.org/solr/guide/6_6/running-solr.html#running-solr) for instructions on how to do that.
+    To help simplify and consolidate the instructions, the important steps are repeated below:
+  
+    > Note: The following instructions assume the installation directory for Solr is `/opt/lucidworks-hdpsearch/solr`.
+  
+    1. Download the [VectorPlugin.jar](https://github.com/saaay71/solr-vector-scoring/blob/master/VectorPlugin.jar) file.
+    
+    2. Create directory `plugins` in the `dist` directory of the Solr installation location.
+    
+    3. Copy VectorPlugin.jar to /opt/lucideworkds-hdpsearch/solr/dist/plugins/
```

Comment:
backtick around VectorPlugin.jar to format it like youi do other files.  The dir too I suppose.